File: data/imported/knowledge/slides/cpp/merged/snippets/snippet_014.cpp
TEST(ConnectorIntegrationTestNoFixture, AdjustmentProperties) {
    Presentation pres;
    auto& conn = pres.slides()[0].shapes().add_connector(
        ShapeType::BENT_CONNECTOR3, 50, 50, 300, 200);
    if (conn.adjustments().size() > 0) {
        auto& adj = conn.adjustments()[0];
        EXPECT_FALSE(adj.name().empty());
        // raw_value is int, angle_value is double — just verify they are accessible
        [[maybe_unused]] int rv = adj.raw_value();
        [[maybe_unused]] double av = adj.angle_value();
    }
}

File: data/imported/knowledge/slides/java/merged/snippets/snippet_014.java
@Test
    void testAddStraightConnectorPersists() throws IOException {
        try (var pres = new Presentation()) {
            pres.getSlides().get(0).getShapes().addConnector(ShapeType.STRAIGHT_CONNECTOR1, 100, 100, 300, 200);

            try (var pres2 = saveAndReopen(pres)) {
                assertThat(pres2.getSlides().get(0).getShapes().size()).isGreaterThanOrEqualTo(1);
            }
        }
    }